modules/n1ql/pages/n1ql-language-reference/alterindex.adoc

```diff
@@ -1,13 +1,16 @@
 = ALTER INDEX
+:page-status: Couchbase Server 5.5
```

Comment:
Presumably the 5.5 refers to its introduction?
Having an orange badge to say so seems confusing.

Reply:
Good point, I'll keep this in for the release/5.5 backport, but remove it from release/6.0 and later.